pkg/content/content.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package contains convenience methods and validator for content-addressed chunks
+package content
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+
+	"github.com/ethersphere/bee/pkg/swarm"
+	bmtlegacy "github.com/ethersphere/bmt/legacy"
+)
+
+// NewContentChunk creates a new content-addressed single-span chunk.
+// The length of the chunk data is set as the span.
+func NewContentChunk(data []byte) (swarm.Chunk, error) {
```

Comment:
Since the package is content, you could also just use a word Chunk instead of ContentChunk, because the usage of exported stuff will be `content.Chunk` or `content.ContentChunk`, choose whatever looks better for you :)